etl/collections/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""This module is a package for collections of indicators. This includes:
```

Comment:
I was trying to think of a better name than collections as it collides with the one from standard library... but I couldn't think of any, so go for it :).

Reply:
yeah, thought that too... can't really think of a better one atm, but im open to rename